### PR TITLE
fix: fix format MCP tool response for null pages in paged_composite result type

### PR DESCRIPTION
**Pull request by hwixley**
## Context
To support null pages in paged_composite MCP result types.

## What
Avoids decoding null pages.

## Have you written unit tests?
- [x] Yes
- [ ] No (add why you have not)

## Are there any specific instructions on how to test this change?

- [ ] Yes (if so provide more detail)
- [ ] No

### CODE REVIEW

File: redbox/tests/api/test_format.py [hunk 1]
```diff
@@ -159,6 +159,52 @@ def test_returns_original_when_no_result_type(self, data):
...
+                {
+                    "result_type": "paged_composite",
+                    "result": {
+                        "items": [
+                            (
+                                {"url": "https://parent.com", "title": "Parent"},
+                                {
+                                    "interactions": {
+                                        "result": {
+                                            "items": [{"url": "https://f.com"}, {"url": "https://g.com"}],
```
Comment (lizzie-perry-dbt) [inline redbox/tests/api/test_format.py:193]: Could add test case for empty items list "items": [] ?

**Review by lizzie-perry-dbt — APPROVED** [review]
Looks great, one minor comment for additional test case, feel free to take or leave

File: redbox/tests/api/test_format.py (hunk 1 above)
Comment (hwixley) [reply redbox/tests/api/test_format.py:193]: good shout added now
